examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -149,10 +149,17 @@ int Commands::Run(int argc, char ** argv)
     return (err == CHIP_NO_ERROR) ? EXIT_SUCCESS : EXIT_FAILURE;
 }
 
-int Commands::RunInteractive(const char * command)
+int Commands::RunInteractive(const char * command, const chip::Optional<char *> & storageDirectory)
 {
     std::vector<std::string> arguments;
     VerifyOrReturnValue(DecodeArgumentsFromInteractiveMode(command, arguments), EXIT_FAILURE);
+    if (storageDirectory.HasValue())
+    {
+        // Propagate the storage directory from the interactive command, so
+        // everything is using the same storage.
+        arguments.push_back("--storage_directory");
```

Comment:
Whey the underscore here? I know that we've got mixed `--foo_bar` and `--foo-bar` convection, but maybe for a new options we can stick to `--foo-bar` (it's easier to type).

Reply:
I initially had it as `--foo-bar`, butA bunch of the other options here seemed to be `--foo_bar`...  I'll change to `--storage-directory`; I agree it's easier to type.

Reply:
Done.